### ISSUE TITLE
project overview: extra filter only partly accessible

### ISSUE THREAD

**Issue by CarolingerSeilchenspringer** [liqd/a4-meinberlin#4045]
**URL:** https://meinberlin-dev.liqd.net/accounts/password/reset/key/2ai-axcbaw-4808e1ce36324459006c8b2f3fa74c43/
**user:** any without mouse
**expected behaviour:** can chose status of participation and project status
**behaviour:** both filter seem to be connected in a way that I cannot chose them individually
**important screensize:**any
**device & browser:** any
**Comment/Question:** 

Screenshot?

**Comment by phillimorland** [comment]
when using with a screen reader this could be much clearer as well e.g.
- serach button needs label
- disconnect the redio fieldsets
- add labelling to them

**Comment by CarolingerSeilchenspringer** [comment]
@phillimorland I will move this into the milestone. As you were mentioning the screen reader maybe a good issue to learn more on testing a11y? maybe @sabinammm and I can have a closer look? Not sure how far I will get, though...

**Comment by phillimorland** [comment]
this is fixed
